File: Src/API/WebApplication5/Model/Puzzel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Model
{
    public class Puzzel
    {
        public int id { get; set; }
        public string Vraag { get; set; }
        public string Antwoord { get; set; }
        public bool juist { get; set; }
       /*   public Puzzel(string omschrijving)
          {
              this.Omschrijving = omschrijving;
          }*/
    }
}

File: MobApp/app/src/main/java/be/ap/eaict/geocapture/Model/Puzzel.java
package be.ap.eaict.geocapture.Model;

public class Puzzel {
    public int id;
    public String vraag;
    public String antwoord;

    public Puzzel(String omschrijving, String antwoord)
    {
        this.vraag = omschrijving;
        this.antwoord = antwoord;
    }
    public Puzzel(int id, String omschrijving, String antwoord)
    {
        this.id = id;
        this.vraag = omschrijving;
        this.antwoord = antwoord;
    }

    public String getVraag() {
        return vraag;
    }

    public String getAntwoord() {
        return antwoord;
    }
}
